convert/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+//! Convertion utilities between protobuf and mongodb
+pub mod mongo;
+pub mod proto;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn cycle_test_mongo() {
```

Comment:
Usually test names are more clear if you name "as an action". For example: "convert_to_mongo_timestamp_test" and "convert_from_mongo_timestamp_test".

I would also break up this test in two unit tests that actually only test one single action each: the first tests systime_to_timestamp and the second tests timestamp_to_systime.
Why? Imagine this test breaks, it is not clear which of the two functions is broken.

Reply:
Then we should test with arbitrary dates, not with the current one, right?

Reply:
Yeah I also think using fixed data is better as it makes the tests more deterministic